### ISSUE TITLE
ACER Failed To Converge for all Events (Trac #170)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#170]
AtmCscdEneryReco fails on all event, even though they passed in earlier versions. I likey induced some bug, if not it has to be documented.

Migrated from https://code.icecube.wisc.edu/ticket/170
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:08:15", 
    "description": "AtmCscdEneryReco fails on all event, even though they passed in earlier versions. I likey induced some bug, if not it has to be documented.", 
    "reporter": "panknin", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547240895681654", 
    "component": "combo reconstruction", 
    "summary": "ACER Failed To Converge for all Events", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2009-10-27T10:10:23", 
    "milestone": "", 
    "owner": "panknin", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20091104T14:40:04: 
* panknin changed status from "new" to "closed"
* panknin commented:

> This are acually several separate things:
> - default settings were dangerous (trying to get direction from seed)
> - module was not complaining setting the status correctly when direction was not set in seed
> - if used with noise the status was used uninitialized
> 
> this was fixed so far.  I had under some circumstances the reconstruction still failing, but this was not reproducible and I am suspecting photonics to be responcable for this. (the fitstatus will be set correctly). I will open a new ticket if it reappears.


* panknin changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20091104T15:47:08: panknin commented:

> Replying to [comment:1 panknin]:
> > - if used with noise the status was used uninitialized
> 
> sorry, this is 
> 
> - if used '''without''' noise the status was used uninitialized

**Comment by icecube-trac** [comment]
Trac update at 20190111T18:54:57: nwhitehorn commented:

> In changeset 167886:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167886"
> Fixes #170 and #137, with new params for Get and some docs
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:21:17: nwhitehorn commented:

> In changeset 168024:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168024"
> Fixes #170 and #137, with new params for Get and some docs
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:08:15: nwhitehorn commented:

> In changeset 168871:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168871"
> Fixes #170 and #137, with new params for Get and some docs
> ```
